### DISCUSSION TITLE
Plans to release the platform-agnostic parts as separate packages?

### DISCUSSION THREAD

**Discussion by benbender** [TanStack/query#2506] [General]
Hey, 

just wanted to know, if there are any Plans to release the platform-agnostic parts of react-query as separate packages in the near(tm) future?

I'm diving my head into svelte lately and would love to have a react-query-like experience there. As I noticed, that most parts of the lib are nicely separated by framework-agnostic/-specific parts, I wondered why there are no separate packages for it so you could easily build on top of it...

**Comment by TkDodo** [comment]
i think we'd need to migrate to a monorepo for that, not sure if Tanner has any plans. 

There is svelte-query, which I think is built on react-query: https://github.com/SvelteStack/svelte-query

**Reply by benbender** [reply]
Yep, saw that. In fact, it's what I saw first and lead to my question here as it did basically fork react-query and hasn't been updated for a while which seems to be missing out newer features and unnecessary work at the same time.

And now I found vue-query which depends on the whole package of react-query and seems to be the smarter way of handling this... But anyways, I think it would be a great idea to migrate to a mono-repo and have all those framework-specific implementations of react-query under one hood, with one community + documentation!

**Reply by TkDodo** [reply]
there is this organization: https://github.com/tanstack

but we'd have to ask @tannerlinsley if his plan is to move the repos there and maybe combine them to a monorepo - I can't tell 🤷
